playback/interception/files/file_interception.py

```diff
@@ -3,15 +3,15 @@
 import os
 import base64
 import sys
-
+import six
 from playback.utils.timing_utils import Timed
 
 _logger = logging.getLogger(__name__)
 
 
 class FileInterception(object):
 
-    ABOVE_LIMIT_CONTENT = 'above interception limit'
+    ABOVE_LIMIT_CONTENT = six.b('above interception limit')
```

Comment:
what's this? why do we need to do anything here?

Reply:
python 3 packages like base64 and zlib compress all use bytes. Instead of passing around strings and then converting to bytes, it makes more sense in python3 to pass around bytes inside these libraries. 

if the "file_content" is bytes in the normal case, then it needs to be bytes in this edge case you added.